invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationList/state/reducer.js

```diff
@@ -9,7 +16,7 @@ export const initialState = {
 
 export default (state = initialState, action) => {
   switch (action.type) {
-    case IS_LOADING:
+    case IS_LOADING || DELETE_IS_LOADING:
```

Comment:
I'm not sure you can you use `||` in a `case` like this. Is it really gonna use `DELETE_IS_LOADING`? Usually fall-through is used when you need to match multiple cases in a `switch`.